tools/devops/device-tests/templates/device-tests.yml

```diff
@@ -246,20 +246,24 @@ steps:
 - bash: |
     set -x
     set -e
+    P=$(echo $XM_STORAGE_PATH | tr -d \'\")
     echo 'Use xamarin storage $USE_XAMARIN_STORAGE'
+    echo 'Use xamarin storage path is $P'
```

Comment:
This will print `Use xamarin storage path is $P'` (and not expand $P).

The same goes for the line just above as well.

Bash doesn't expand variables within single-quoted strings.

Reply:
+1, single quotes are for strings w/o var expansion. Do things get expanded in DevOps?